Resources/Maps/Misc/pocket_dimension.yml

```diff
@@ -0,0 +1,210 @@
+meta:
+  format: 4
+  name: Pocket Dimension
+  author: Ilya246
+  postmapinit: false
+tilemap:
+  0: Space
+  55: FloorPlanetGrass
+entities:
+- proto: ""
+  entities:
+  - uid: 1
+    components:
```

Comment:
Blacklist construction so people can't build inside the dimensions

Reply:
is there just a component added to the BecomesStation entity?

Reply:
being able to build inside of them is intentional and i, again, don't see how it could cause issues (and if it could, i'd argue the 30TC cost justifies it)

Reply:
It's for balancing reasons. This is required for this to be merged, please add it.